lib/matplotlib/tests/test_ticker.py

```diff
@@ -725,6 +726,24 @@ def test_mathtext_ticks(self):
             ax.set_xticks([-1, 0, 1])
             fig.canvas.draw()
 
+    def test_cmr10_substitutions(self, caplog):
+        mpl.rcParams.update({
+            'font.family': 'cmr10',
+            'mathtext.fontset': 'cm',
+            'axes.formatter.use_mathtext': True,
+        })
+
+        # Test that it does not log a warning about missing glyphs.
+        with caplog.at_level(logging.WARNING, logger='matplotlib.mathtext'):
+            fig, ax = plt.subplots()
+            ax.plot([-0.03, 0.05], [40, 0.05])
+            ax.set_yscale('log')
+            yticks = [0.02, 0.3, 4, 50]
+            formatter = mticker.LogFormatterSciNotation()
+            ax.set_yticks(yticks, map(formatter, yticks))
```

Comment:
If the ticks are not specified, it draws the ticks at 0.01, 0.1, 1, 10, etc. (so the multiplication sign never appears). Specifying them works. I verified that the test fails if either of the minus and multiplication signs are not found.